### PR TITLE
Add nightly builds via GitHub Actions 

### PR DESCRIPTION
**Pull request by JRoy**
### Description
This pull request utilizes GitHub's own `upload-artifact` action to upload the Skript build of each commit/pull request into the workflow results page. 

Here is a picture of where builds are found: 
Capture

Sadly, it's a zip file with only one file in it for now. This is a limitation of the `upload-artifact` action and requires GitHub to open source this action to be fixed but really doesn't matter for the time being.

### CODE REVIEW

**Comment by bensku** [comment]
Where exactly the builds go? I don't want to clutter this repository's releases.

**Comment by JRoy** [comment]
@bensku GitHub stores it some backend, not the releases tab

**Comment by bensku** [comment]
Sounds good! I'll do some build engineering when merging this to get build metadata into created builds.

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -17,5 +17,10 @@ jobs:
         java-version: 1.8
     - name: Build Skript
       run: ./gradlew build
+    - name: Upload Artificat
```
Comment (TheDGOfficial) [inline .github/workflows/gradle.yml]: small typo

```suggestion
    - name: Upload Artifact
```

Comment (JRoy) [reply .github/workflows/gradle.yml]: Thanks :p

**Comment by bensku** [comment]
I'll do this a bit differently to avoid getting nightly builds that fail tests downloadable. Thanks for great idea, though!

**Pull request closed without merging**